### ISSUE TITLE
Suggestion for a new API

### ISSUE THREAD

**Issue by shantamsultania** [mishraaditya595/ScanIt#68]
Hey, I have a feature suggestion basically I wanted to add an API for you in which you can add all the Scans manually it thus taking your dependency from other third party software to your own this is just a suggestion that's why I am not adding it to your existing android application just suggesting it so that it could be of future use.

**Comment by mishraaditya595** [comment]
I would like to here more about the API

**Comment by shantamsultania** [comment]
Ok so in Python we have OpenCV that can do the task that you are doing that is to Scan and convert easyly right know you capture the Image and sends it to other reference APIS/ lib of Firebase be it ML kit or more this makes you a bit 3rd party dependent 
I am thinking of making an API for you that accepts an image as parameter and returns you a file required by you be it .pdf and all 
same I want to do it for Scan

**Comment by mishraaditya595** [comment]
Sounds pretty exciting to me. You can start working on it. Are you a part of GSSOC?

**Comment by shantamsultania** [comment]
yeah, will start working on making a conversion API for the image to other supported formats first after that a scanner image 
also yeah I am a part of GSSOC

**Comment by shantamsultania** [comment]
hey @mishraaditya595 can you can also add the level and assigned label to the same

**Comment by mishraaditya595** [comment]
Tell me one thing @shantamsultania, once the API is done, are you going to integrate it with the app?

**Comment by shantamsultania** [comment]
right now no as you have a large number of functionalities in the applications and using OpenCV SDK also so will be first creating  an API and sharing you for review (also I will be sharing a code of how to integrate it with the Android app )
if you like the API will do the interaction as a part of the new task.

**Comment by mishraaditya595** [comment]
ok go ahead

**Comment by mishraaditya595** [comment]
Hey @shantamsultania, it's been almost a month and I am yet to hear from you.
